3p/integration.js

```diff
@@ -360,8 +361,8 @@ function triggerResizeRequest(width, height) {
   nonSensitiveDataPostMessage('embed-size', {width, height});
 }
 
-function triggerRenderStart() {
-  nonSensitiveDataPostMessage('render-start');
+function triggerRenderStart(width, height) {
```

Comment:
I copy the format from other function that postMessages. `triggerRenderStart()` and `triggerRenderStart(width, height)` both work, how can we represent this in our code and let 3p ad know?